CHANGELOG.md

```diff
@@ -4,6 +4,129 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
 
 ## Unreleased
 
+## v0.7.0-rc2 - 2023-01-18
+
+The documentation for this version is available at the [Earthly 0.7 documentation page](https://docs.earthly.dev/v/earthly-0.7/).
+
+**Earthly CI**
+
+Earthly 0.7 is the first version compatible with Earthly CI.
+
+Earthly 0.7 introduces the new keywords `PIPELINE` and `TRIGGER` to help define Earthly CI pipelines.
+
+```
+my-pipeline:
+    PIPELINE --push
+    TRIGGER push main
+    TRIGGER pr main
+    BUILD +my-target
+```
+
+For more information on how to use `PIPELINE` and `TRIGGER`, please see the [reference documentation](https://docs.earthly.dev/v/earthly-0.7/docs/earthfile#pipeline-beta).
+
+**Podman support**
+
+Podman support has now been promoted out of *beta* status and is generally available in 0.7. Earthly will automatically detect the container frontend, whether that's `docker` or `podman` and use it automatically for running Buildkit locally, or for outputting images locally resulting from the build.
+
+Please note that rootful podman is required. Rootless podman is not supported.
+
+**VERSION is now mandatory**
+
+The `VERSION` command
+is now required for all Earthfiles, and an error will occur if it is missing. If you are not ready to update your
+Earthfiles to use 0.7 (or 0.6), you can declare `VERSION 0.5` to continue to use your Earthfiles.
+
+**Pushing no longer requires everything else to succeed**
+
+The behavior of the `--push` mode has changed in a backwards incompatible manner. Previously, `--push` commands would only execute if all other commands had succeeded. This precondition is no longer enforced, to allow for more flexible push ordering via the new `WAIT` clause. To achieve the behavior of the previous `--push` mode, you need to wrap any pre-required commands in a `WAIT` clause. For example, to push an image only if tests have passed, you would do the following:
+  
+```Earthfile
+test-and-push:
+  WAIT
+    BUILD +test
+  END
+  BUILD +my-image
+my-image:
+  ...
+  SAVE IMAGE --push my-org/my-image:latest
+```
+
+or
+
+```Earthfile
+test-and-push:
+  FROM +some-image
+  WAIT
+    BUILD +test
+  END
+  SAVE IMAGE --push my-org/my-image:latest
+```
+
```

Comment:
It might be helpful to additionally include a CD-type use-case here.

e.g.

The `WAIT` clause can also be used to first push an image, then run a command that deploys that image on a remote server, for example

```
my-image:
  ...
  SAVE IMAGE --push my-org/my-image:latest

push-and-deploy:
  FROM +some-image
  WAIT
    BUILD +my-image
  END
  RUN --push ./deploy my-org/my-image:latest
```

Reply:
Good idea - added!